visualization/simulate/index.js

```diff
@@ -0,0 +1,98 @@
+const aws = require('aws-sdk');
```

Comment:
This is really great for end to end testing: it's already helped me catch at least two bugs. For that purpose, would love to see this simulator support chills as well.

(Non-blocking, obviously. I can work on adding this next week if you want.)